lib/pf/Authentication/CreateLocalAccountRole.pm

```diff
@@ -0,0 +1,41 @@
+package pf::Authentication::CreateLocalAccountRole;
+
+=head1 NAME
+
+pf::Authentication::CreateLocalAccountRole
+
+=head1 DESCRIPTION
+
+Role that defines the Authentication source behavior for creating local accounts
+
+=cut
+
+use Moose::Role;
+
+has 'create_local_account' => (isa => 'Str', is => 'rw', default => 'no');
+has 'local_account_logins' => (isa => 'Str', is => 'rw', default => 0);
```

Comment:
I'd use a constant (constant is the key :tm: ) for the '0' I hate to grep for '0' if you know what I mean ;)